framework/files/etc/cos/bootargs.cfg

```diff
@@ -1,7 +1,7 @@
 set kernel=/boot/vmlinuz
 if [ "${img}" == "/cOS/recovery.img" ]; then
-  set kernelcmd="console=tty1 root=LABEL=$recovery_label cos-img/filename=$img rd.neednet=0 rd.cos.oemlabel=$oem_label rd.cos.mount=LABEL=$oem_label:/oem"
+  set kernelcmd="console=tty1 root=LABEL=$recovery_label elemental.image=recovery rd.neednet=0 elemental.oemlabel=$oem_label"
 else
-  set kernelcmd="console=tty1 root=LABEL=$state_label cos-img/filename=$img panic=5 rd.neednet=0 rd.cos.oemlabel=$oem_label fsck.mode=force fsck.repair=yes rd.cos.mount=LABEL=$oem_label:/oem rd.cos.mount=LABEL=$persistent_label:/usr/local"
+  set kernelcmd="console=tty1 root=LABEL=$state_label elemental.image=$mode panic=5 rd.neednet=0 elemental.oemlabel=$oem_label fsck.mode=force fsck.repair=yes"
```

Comment:
where does $mode come from ?

Reply:
It's a new variable coming from grub.cfg in the toolkit. I realized it won't be backwards compatible tho, so I changed it back for active/passive.